package.json

```diff
@@ -26,6 +26,7 @@
     "postcss-loader": "^1.3.3",
     "react": "^15.5.4",
     "react-dom": "^15.5.4",
+    "redux-entity-config": "^1.0.0",
```

Comment:
I found this a little confusing to use when I first started on dispatch, probably because I was so consumed with just understanding react/redux. Definitely want to use it in our apps, but I wonder if it's going to be instructive to have new devs do it manually for the first couple entities, then refactor to use this. Now that I say it that seems like unnecessary churn, but maybe there's a way to help instruct our team on the concept of entities before we explain this.